python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -40,6 +41,18 @@ class SGDBackendError(Exception):
     """Errors with BackendExecutor that should not be exposed to user."""
 
 
+class PersistedCheckpoint:
+    def __init__(self, path, priority):
+        self.path = path
+        self.priority = priority
+
+    def __lt__(self, other):
+        return self.priority < other.priority
+
+    def __repr__(self):
+        return f"PersistedCheckpoint({repr(self.path)})"
+
+
 class CheckpointManager:
```

Comment:
Should we move this and `PersistentCheckpoint` into the `checkpoint` module? This is starting to contain more logic so maybe we want to separate them from `backend`.